gdi/get-data-in/connect/gcp/gcp-connect.rst

```diff
@@ -98,8 +98,17 @@ Your GCP integration is now complete.
 
 .. note:: Splunk is not responsible for data availability, and it can take up to several minutes (or longer, depending on your configuration) from the time you connect until you start seeing valid data from your account. 
 
-Options
-++++++++
+Using a single principal for your resources
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+In IAM you can grant access to your resources to one or more entities called principals. The principal provided for the project needs either the ``serviceusage.services.use`` permission or the Service Usage Consumer role.
```

Comment:
``serviceusage.services.use`` permission or the Service Usage Consumer role - this is only required when using :strong:`Use quota from the project where metrics are stored or syncing cloudsql metadata

Reply:
Ah my bad, fixing it!

Reply:
@anosek-an corrected!